code/languages/cpp/interval_tree.cpp

```diff
@@ -0,0 +1,101 @@
+
```

Comment:
just add the comment as mentioned in [contribution guide](https://github.com/OpenGenus/cosmos/wiki/contribute)